spring/src/main/java/org/axonframework/spring/config/RepositoryFactoryBean.java

```diff
@@ -28,6 +28,7 @@
   * @author Allard Buijze
   * @since 3.3
   */
+ @Deprecated
```

Comment:
This change warrants an adjustment in the JavaDoc describing why it's deprecated and what (if any) replaces it.